POC-OOBE/OOBE-Sandbox/PowerToys Settings Sandbox/Views/MainPage.xaml.cs

```diff
@@ -13,6 +32,124 @@ public sealed partial class MainPage : Page
         public MainPage()
         {
             InitializeComponent();
+            powerOnLaunchDialog();
+        }
+
+        private async void powerOnLaunchDialog()
```

Comment:
Can you create a new class for the OnLaunch dialog, and call it in main page, instead of including it directly in main page?

Reply:
If you create a public method 
'''
        public void BeginSettingsTips()
        {
            OpenFirstGeneralSettingsTip();
        }
'''
in the mainpage.xaml.cs, you should be able to call the settings tips outside the mainpage and put your popup as its own component

Reply:
Great suggestion. will work on it